vllm/attention/backends/rocm_flash_attn.py

```diff
@@ -459,10 +453,12 @@ def __init__(
         if blocksparse_params is not None:
             raise ValueError(
                 "ROCmFlashAttention does not support blocksparse attention.")
-        if logits_soft_cap is not None:
-            raise ValueError(
-                "ROCmFlashAttention does not support attention logits soft "
-                "capping.")
+
+        if logits_soft_cap is None:
+            # In flash-attn, setting logits_soft_cap as 0 means no soft cap.
+            logits_soft_cap = 0
```

Comment:
This effectively enables logits_soft_cap for any model, unconditionally.

Reply:
In flash attention, it is 0 by default, I think. https://github.com/ROCm/flash-attention/blob/main/flash_attn/flash_attn_interface.py#L1334